### ISSUE TITLE
Script to replace names and abstracts in person records

### ISSUE THREAD

**Issue by davidamichelson** [srophe/srophe-app-data#293]
I would like to insert these:https://github.com/srophe/persons/blob/master/new-saints-data/incomplete/abstracts-incomplete-saints.xml back into their records. Could you please write a script, they names and notes in this file should replace the existing ones.

**Comment by davidamichelson** [comment]
Okay, my edits are done for now at least. I did discover there are still records that don't have xml:lang="en" names, will that be a problem?

For the time being, could you please insert these: https://github.com/srophe/persons/blob/master/new-saints-data/incomplete/abstracts-incomplete-saints.xml

Could you please insert them back into the srophe/persons//incomplete/tei files? The tei:note @type=abstract should replace the abstracts in each doc, no ids to match on but there is only one note of that type per doc. The perseNames should replace their previous versions based on their @xml:id, but should not replace all the names, just the ones that match based on xml:id

Please do the insert on a branch of persons repo so I can check it.

Thanks,
DM

**Comment by davidamichelson** [comment]
Oh, if it's not too late here is the revisionDesc statement to include in each of these records:

```
<change who="http://syriaca.org/documentation/editors.xml#dmichelson" n="2.0" when="2016-6-24-09:00">Change: tagged abstract for persons, places, titles and edited the "en" persNames</change>
```

**Comment by wsalesky** [comment]
Okay... just writing it right now. Good timing.

**Comment by wsalesky** [comment]
@davidamichelson Do you want me to replace the persNames as well? It overwrites the headwords script. (Although I guess I can just rerun that after I run this one. )

**Comment by davidamichelson** [comment]
Ah, yes, to clarify further please replace the persNames as I edited them in that file, and then run the headwords script to chose the [1] first "en" persName as headword. But please only run this on the incompletes! I have started to edit the headwords on the other saints.

Thanks

**Comment by wsalesky** [comment]
Okay. Here is the updated data... on a new branch:
https://github.com/srophe/persons/tree/issuepersons293

**Comment by davidamichelson** [comment]
hmm, I am not showing that it worked, are you sure?

**Comment by wsalesky** [comment]
Hum, looks like the changes did not actually get pushed... let me try again.

**Comment by wsalesky** [comment]
Sorry about that, forgot to move the data to the correct place. :)
